### DISCUSSION TITLE
Menu icon discrepancies

### DISCUSSION THREAD

**Discussion by emorywolfe** [blish-hud/Blish-HUD#928] [Blish HUD Core Ideas]
![blishOverlayMenuIcons](https://github.com/blish-hud/Blish-HUD/assets/41130701/704bbd8e-bdcb-4e5c-b978-99f730a5b985)

At the moment I have my GW2 interface size set to normal, & you can see in the attached picture that if I set Blish's interface size to normal to match, the icons are pushed too far to the left & too high. If I set the Blish interface size to large, it looks a bit more closer to the size of the regular icons, however now there is a big gap in between all the icons, & they are pushed too far down.

I absolutely love Blish, but I feel it looks a bit unpolished with these placement issues. Is there a way to possibly fix the placement, either with Blish' interface size set to match the GW2 settings or on large? Either would be great, just think it would look a lot cleaner if things were aligned a bit more.

Thanks for all the hard work,
- Hyper

**Comment by Taschenbuch** [comment]
Related discord discussion:
https://discord.com/channels/531175899588984842/534490472224391169/1169932193997520916

I personally think that this is just a visual effect caused by the wizard and lion icons. They exend further down than the rest of the icons. And the wizard icon is visually big compared to the other gw2 icons. 

screenshot shows examples of adding an x,y offset of 1,1 or 2,2 to the CornerIcon. It is hard to see in the screenshot, but already the 2,2 offset looks wrong. Blish icon would be too far down.
![image](https://github.com/blish-hud/Blish-HUD/assets/43114787/9fbb6e26-a87c-4891-90b2-550c5abab23f)

using a bigger blish interface size than the gw2 size, makes the blish icon more equal to wizard and lion icon than to the other icons.
![image](https://github.com/blish-hud/Blish-HUD/assets/43114787/63bae61e-54d6-4941-9fdf-e70bcc573145)

**Reply by emorywolfe** [reply]
1,1 looks absolutely perfect

**Comment by dlamkins** [comment]
We in the past have already done 1,1 movements to try and make somebody else happy that was annoyed with the pixel offset. 

Regarding the gap, if there is a desync in UI size (normal / large) there is going to be a gap. I won't be supporting desynced sizes and trying to realign stuff as it affects far more than just the corner icon.

**Reply by emorywolfe** [reply]
ah yes, the desync thing makes sense. according to the screenshot @Taschenbuch posted, though, it looks like one final 1,1 movement would make it flawless. i've had a few guildies that say it does look a bit off, since we see it constantly, so I know I'm not the only one aha, but up to you guys of course. regardless thank you so much for all the hard work you all have put into this amazing add on <3
